### PR TITLE
Support sync stream id

### PR DESCRIPTION
**Pull request by cnderrauber**

### CODE REVIEW

File: .changeset/soft-terms-wait.md
Comment (davidzhao) [inline .changeset/soft-terms-wait.md:1]: I think we only need one of the changesets.

File: src/room/participant/LocalParticipant.ts [hunk 1]
```diff
@@ -652,6 +652,8 @@ export default class LocalParticipant extends Participant {
       encryption: this.encryptionType,
       stereo: isStereo,
       disableRed: this.isE2EEEnabled || !(opts.red ?? true),
+      stream: opts?.stream,
+      // disableRed: !(opts.red ?? true),
```
Comment (davidzhao) [inline src/room/participant/LocalParticipant.ts:656]: remove `// disableRed` line?

File: src/room/track/options.ts [hunk 2]
```diff
@@ -109,6 +109,13 @@ export interface TrackPublishOptions extends TrackPublishDefaults {
    * Source of track, camera, microphone, or screen
    */
   source?: Track.Source;
+
+  /**
+   * set stream name of the track belongs to, a/v tracks belong to a same stream
+   * will be synchronized better than standalone tracks. if left empty, the server
+   * will try infer it from the track source.
+   */
```
Comment (davidzhao) [inline src/room/track/options.ts:117]:
```suggestion

  /**
   * Set stream name for the track. Audio and video tracks with the same stream name
   * will be placed in the same `MediaStream` and offer better synchronization.
   * By default, camera and microphone will be placed in a stream; as would screen_share and screen_share_audio
   */
```

**Review by davidzhao — APPROVED** [review]
lg, just a couple of nits around docs.

**Review by davidzhao — APPROVED** [review]
lg, just a couple of nits around docs.

**Comment by cnderrauber** [comment]
Will hold this PR until all the `syncStream` flag of sfu is released.

**Pull request merged**